app/scripts/views/mixins/session-verification-poll-mixin.js

```diff
@@ -96,7 +96,7 @@ define(function (require, exports, module) {
         // if the user is switching networks.
         // Report a known error to Sentry, but not the user.
         // Details: github.com/mozilla/fxa-content-server/issues/2638.
-        this.logError(AuthErrors.toError('POLLING_FAILED'));
+        this.logError(AuthErrors.toError('POLLING_FAILED', JSON.stringify(err)));
```

Comment:
@seanmonstar - I'm having a hard time forcing this path to be taken, and I suspect there is a problem here - by time `err` is caught here, it's already too late - `err` is no longer the XHR response. fxa-client.js has [already converted the response to an AuthError based error](https://github.com/mozilla/fxa-content-server/blob/polling-failed-details/app/scripts/lib/fxa-client.js#L71), and done so badly in some cases. For example, if I have the fxa-js-client library's recoveryEmailStatus function reject with `{status: 300}` object, by time the error arrives here, it's mangled. The error's `errno` field is an object that contains `status: 300`:

<img width="390" alt="screen shot 2017-08-17 at 16 44 20" src="https://user-images.githubusercontent.com/848085/29420988-304562da-8363-11e7-9968-85b12cf75e9d.png">

This happens because of the way [the response object is converted](https://github.com/mozilla/fxa-content-server/blob/polling-failed-details/app/scripts/lib/errors.js#L119). `toErrno` looks for an `errno` inside the object, if one doesn't exist, it returns the original object. An `Error` object is created, and it's `errno` property is set to the object that was passed in.

The branch you [would expect to be called](https://github.com/mozilla/fxa-content-server/pull/5348/files#diff-739ad3229f46c545f813584415575731R99) is passed over, instead, [displayError is called below](https://github.com/mozilla/fxa-content-server/pull/5348/files#diff-739ad3229f46c545f813584415575731R106), and when the error is reported to the content-server, it blows up with a validation error: 

![screen shot 2017-08-17 at 16 51 33](https://user-images.githubusercontent.com/848085/29421237-f57c0da6-8363-11e7-8a51-d5c0ac7e94f9.png)

I made horrible mistakes with those error modules, trying to be too clever. I'm going to put some thought into what `toError` should do in this case, certainly not what it's doing.